README.md

```diff
@@ -50,6 +50,8 @@ This returns a list of matching documents with a score of how closely they match
 
 [API documentation](https://lunrjs.com/docs/index.html) is available, as well as a [full working example](https://olivernn.github.io/moonwalkers/).
 
+In Depth Guide: https://lunrjs.com/guides/upgrading.html
```

Comment:
I think linking to the [getting started](https://lunrjs.com/guides/getting_started.html) guide is probably better.